internal/tsc_wrapped/cache.ts

```diff
@@ -127,7 +127,7 @@ class Cache<T> {
   }
 }
 
-interface SourceFileEntry {
+export interface SourceFileEntry {
```

Comment:
interface needs exporting as its used as a parameter in an exported class

Reply:
true for ts 2.7 at least. this is no longer an error as of ts 2.9. didn't check 2.8.